components/title/title.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+//@ts-ignore
+import styles from './title.module.scss';
+
+/**
+ * @typedef {Object} TitleProps
+ * @property {string} text - the text that would be in the title
+ * @property {string} [emoji] - emoji to animate the title
+ */
+
+/**
+ * 
+ * @param {TitleProps} props
+ */
+
+export const Title = ({ text, emoji }) => {
+  return (
+    <h1 className={styles.title}>{text} {emoji}</h1>
```

Comment:
Wrapped the emoji with a span an added a class to style it. in the xd it has a separation with the text of 24px.

Reply:
Done!